include/nlohmann/json.hpp

```diff
@@ -1227,13 +1233,70 @@ class basic_json
     invariant. Furthermore, it has to be called each time the type of a JSON
     value is changed, because the invariant expresses a relationship between
     @a m_type and @a m_value.
+
+    Furthermore, the parent relation is checked for arrays and objects: If
+    @a check_parents true and the value is an array or object, then the
+    container's elements must have the current value as parent.
+
+    @param[in] check_parents  whether the parent relation should be checked.
+               The value is true by default and should only be set to true
```

Comment:
...only be set to `false`.

Reply:
Good catch! Fixed in a77621687a826bfb020a3ceeda656f7b45479b44.